README.md

```diff
@@ -80,3 +80,5 @@ _In development on the_ [_`web` branch_](https://github.com/msarmi9/Sparkle/tree
 ![webapp](./docs/images/web_app.png)
 
 _This is a screenshot of a drafted UI and is likely to change in the coming weeks._
+
+Our project board may be found [here](https://github.com/msarmi9/Sparkle/projects/5)
```

Comment:
@ajcheon Maybe you merge `classroom/master` into your local `web` branch before submitting this PR? Just a guess b/c I added this line to the `README.md` on the `classroom` repo using the GitHub GUI.